Bottom: bd324713cdc681e6d486ab7e7f253e99b4f78db6
Top:    12180faebfa57cd6f3feeeab4c2b416e099c7a92
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-06-12 13:16:09 -0400




---

diff --git a/arch/arm/boot/dts/sun7i-a20-cubietruck.dts b/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
index 365e206..bed74a9 100644
--- a/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
+++ b/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
@@ -77,6 +77,18 @@
 			status = "okay";
 		};
 
+		spi2: spi@01c17000 {
+			pinctrl-names = "default";
+			pinctrl-0 = <&spi2_pins_a>;
+			status = "okay";
+
+			dac0: dh2228@2 {
+				compatible = "rohm,dh2228fv";
+				reg = <2>;
+				spi-max-frequency = <100000>;
+			};
+		};
+
 		pinctrl@01c20800 {
 			mmc3_pins_a: mmc3@0 {
 				/* AP6210 requires pull-up */
